content/news/2024-09-08/405/index.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "2024.09.08 release has been sent to review"
+date: 2024-09-08T15:08:10+00:00
+slug: "2024-09-08-release-has-been-sent-to-review"
+taxonomies:
+  news: ["Releases"]
+---
+
+2024.09.08 release has been sent to review.
+- **Track Recorder:** Record and save your routes while hiking, biking, running, or navigating.
+- **OpenStreetMap:** Swithched to browser-based authentication to comply with google play on Android.
+- **New Map Data:** Updated with data as of September 7, including isolines for Egypt and Algeria.
+- **Translations:** Enhanced Chinese, Hebrew, and Italiand, and added translations for trail difficulty.
+
+Get the new release first before it gets approved Apple/Google:
+- iOS:  [TestFlight](https://testflight.apple.com/join/lrKCl08I)
+- Android: see APKs attached in Telegram channel
+
+Detailed changes can be seen on [GitHub](https://github.com/organicmaps/organicmaps/releases/tag/2024.09.08-7-android).
```

Comment:
Github release page is not detailed, but it refers to the site for the detailed changelog itself :)

Actually, we are supposed to have the best-detailed changelog on-site.
Maybe I can take a look later and will it with more info based on the commits range ..